### PR TITLE
HTML Assignment - Level 1

### PR DESCRIPTION
**Pull request by avantighatage**
`HTML` Assignment - Level 1
Please review it.

### CODE REVIEW

File: css/dashboard.css [hunk 1]
```diff
@@ -0,0 +1,71 @@
+.headerlogo{
```
Comment (mehulcse) [inline css/dashboard.css]: Please remove the empty class.

File: css/dashboard.css [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+}
+.btnimages{
+
+	   float: right;
+    margin-top: -30px;
+}
+table {
+    font-family: arial, sans-serif;
+   border-collapse: collapse;
+
```
Comment (mehulcse) [inline css/dashboard.css]: Remove empty space.

File: css/dashboard.css [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+td, th {
+    border-bottom: : 0px solid #dddddd;
+    text-align: center;
+    padding: 8px;
+}
+tr.border_bottom  {
+  border-bottom:1pt solid black;
+}
+.btncontinue
```
Comment (mehulcse) [inline css/dashboard.css]: Indentation issue.

**Review by mehulcse — CHANGES_REQUESTED** [review]
Kindly fix the indentation issue.

**Comment by avantighatage** [comment]
Fixed code review comment.

**Review by mehulcse — APPROVED** [review]

**Pull request merged**